hugetlb.go

```diff
@@ -13,7 +13,7 @@ import (
 func NewHugetlb(root string) (*hugetlbController, error) {
 	sizes, err := hugePageSizes()
 	if err != nil {
-		return nil, nil
+		return nil, err
 	}
```

Comment:
Hmm..I meant to make this non-critical as well, but now I see that the `defaults()` setup errors out instead of just not including hugetlb subsystem. So at this point it is panic or failure.. let me try and update the PR